test/rekt/features/sinkbinding/feature.go

```diff
@@ -76,6 +77,54 @@ func SinkBindingV1Deployment(ctx context.Context) *feature.Feature {
 	return f
 }
 
+func SendsEventsWithBrokerAsSinkTLS(ctx context.Context) *feature.Feature {
+	sbinding := feature.MakeRandomK8sName("sinkbinding")
+	brokerName := feature.MakeRandomK8sName("broker")
+	sinkName := feature.MakeRandomK8sName("sink")
+	extensionSecret := string(uuid.NewUUID())
+
+	f := feature.NewFeatureNamed("SinkBinding V1 Deployment BrokerAsSink test")
+	f.Prerequisite("transport encryption is strict", featureflags.TransportEncryptionStrict())
+	f.Prerequisite("should not run when Istio is enabled", featureflags.IstioDisabled())
+
+	f.Setup("install broker", broker.Install(brokerName, broker.WithEnvConfig()...))
```

Comment:
For this line to work you will need to import the broker rekt package by adding `"knative.dev/eventing/test/rekt/resources/broker"` to the list of imports in this file